### PR TITLE
Adds Trigger for SauceLabs Tests Using Travis

### PR DESCRIPTION
**Pull request by willmichael**
Updates settings and adds new configuration file which is properly configured for the travis vm.
Kicks off saucelabs (automated browser tests) through travis when a PR is opened or a commit is added to master. 

This PR does not address needed testing changes.

### CODE REVIEW

File: travis-karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      version: '68.0',
+      platform: 'OS X 10.12'
+    },
+    SauceLabsFirefox: {
+      base: 'SauceLabs',
+      browserName: 'firefox',
+      version: '64.0',
+      platform: 'Windows 10'
+    }
+  }
```
Comment (galxy25) [inline travis-karma.conf.js]: Can we pin  the browser to latest(https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options)(that way as browsers update our tests are testing on those new browsers and letting us knew when the browsers broke something without us having to manually update these versions), and if this works can you update it to test both chrome & Firefox on windows & os x?

Comment (willmichael) [reply travis-karma.conf.js]: Yup, I don't see why not

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Tozny Storage (TozStore) Browser Sodium SDK (JavaScript For The Browser)
 
+[![Build Status](https://travis-ci.org/tozny/browser-sodium-sdk.svg?branch=master)](https://travis-ci.org/tozny/browser-sodium-sdk)
```
Comment (lkwdwrd) [inline README.md]: Yay!

File: karma.conf.js [hunk 3]
```diff
@@ -1,18 +1,30 @@
...
-    },
-    SauceLabsFirefox: {
-      base: 'SauceLabs',
-      browserName: 'firefox',
-      version: '64.0',
-      platform: 'Windows 10'
+      version: 'latest',
+      platform: 'macOS 10.12'
     }
+    // SauceLabsFirefoxWindows: {
```
Comment (lkwdwrd) [inline karma.conf.js]: Why are these commented?

Comment (willmichael) [reply karma.conf.js]: I was doing some iterative testing and this commit slipped through; it will be fixed in the next PR

**Comment by willmichael** [comment]
> Is this runnable outside of sauce labs, or does running the tests locally mean you need your browser open to the sauce labs API to see the video/result?

@lkwdwrd the results are available locally within the terminal you ran them in, but they do require you to export your saucelabs user/apikey as per the older development.md doc

**Pull request merged**